container_api/src/holochain.rs

```diff
@@ -87,17 +87,13 @@ impl Holochain {
         let mut instance = Instance::new(context.clone());
         let name = dna.name.clone();
         instance.start_action_loop(context.clone());
-        let context = instance.initialize_context(context.clone());
-        let context2 = context.clone();
-        let result = block_on(
-            initialize_application(dna, &context2).and_then(|_| initialize_network(&context)),
-        );
+        let result = block_on(network::initialize(&instance, Some(dna), context.clone()));
```

Comment:
Yeah, I was thinking this misses `initialize_application` but that is included in this workflow. But `network::initialize` is misleading then.